src/abyssal_tome/constants.py

```
@@ -0,0 +1,99 @@
+# Old data paths (relative to project root, assuming scripts run from root or handle paths appropriately)
+OLD_PLAYER_CARDS_PATH = PROJECT_ROOT / "player_cards.json"
+OLD_OTHER_CARDS_PATH = PROJECT_ROOT / "other_cards.json"
+OLD_FAQS_DIR_PATH_PROCESS_JSON = FAQS_DIR # For process_json.py that globs this dir
+
+
+# --- Constants for scripts/process_new_format.py ---
+RULING_REMOVAL_PATTERNS: list[str] = [
+    "FAQ removed - double-checking provenance.",
+    "OVERRULED SEE BELOW",
+    "SEE BELOW",
+    'Matt writes: "This was unintentional and we are looking into fixing this, perhaps in the next edition of the FAQ."',
```

Ruff (0.11.9)

Line too long (121 > 100)

(E501)